### PR TITLE
Define "Permissions-Policy" as a structured header

### PR DESCRIPTION
**Pull request by clelland**
Closes: #376

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -80,10 +94,9 @@ spec:reporting; urlPrefix: https://w3c.github.io/reporting/
     within their application. It can do so by delivering the following HTTP
     response header to define a permissions policy:</p>
     <pre>
-      <a http-header>Permissions-Policy</a>: fullscreen 'none'; geolocation 'none'</pre>
-    <p>By specifying the "<code>'none'</code>"keyword for the origin list, the
-    specified features will be disabled for all browsing contexts, regardless of
-    their origin.</p>
+      <a http-header>Permissions-Policy</a>: fullscreen=(), geolocation=()</pre>
+    <p>By specifying an empty origin list, the specified features will be
+    disabled for all browsing contexts, regardless of their origin.</p>
```
Comment (annevk) [inline index.bs:99]: documents (browsing contexts don't have an origin)

File: index.bs [hunk 2]
```diff
@@ -743,14 +782,31 @@ partial interface HTMLIFrameElement {
     <div class="algorithm" data-algorithm="parse-header">
     Given a string (|value|) and an [=origin=] (|origin|), this algorithm will
     return a <a>declared permissions policy</a>.
+    1. Let |parsed header| be the result of parsing |policy| as a dictionary.
```
Comment (annevk) [inline index.bs:785]: Where does policy come from? Also, I think this algorithm should take a SH dictionary. And elsewhere we'd use https://fetch.spec.whatwg.org/#concept-header-list-get-structured-header to get it from a header list.

Comment (clelland) [reply index.bs:785]: Oops -- |policy| should have been |value| here.

And we can take a dictionary and defer to fetch for the SH parsing, thanks.

File: index.bs (hunk 1 above)
Comment (clelland) [reply index.bs]: Fixed, thanks.

**Pull request merged**